src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
Thinking more about this, there is only ```Rgb<SrgbPrim<D65>, D65>``` or ```Rgb<SrgbPrim<D50>, D50>```. ```Rgb<SrgbPrim<D65>, D50>``` does not exists, but what we are trying to say  is that we have know values of the Srgb primaries which can be converted into ```Rgb<SrgbPrim<D50>, D50>``` via chromatic adaptation. Does that make sense? When we change the white point everything including the primaries converts into the new white point is my understanding